jbpm-services/jbpm-kie-services/src/test/java/org/jbpm/kie/services/test/ProcessServiceWithEntitiesTest.java

```diff
@@ -0,0 +1,165 @@
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jbpm.kie.services.test;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.drools.compiler.kie.builder.impl.InternalKieModule;
+import org.drools.core.util.IoUtils;
+import org.drools.persistence.jpa.marshaller.VariableEntity;
+import org.hibernate.LazyInitializationException;
+import org.jbpm.kie.services.impl.KModuleDeploymentUnit;
+import org.jbpm.kie.test.util.AbstractKieServicesBaseTest;
+import org.jbpm.runtime.manager.impl.deploy.DeploymentDescriptorImpl;
+import org.jbpm.services.api.model.DeploymentUnit;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.kie.api.KieServices;
+import org.kie.api.builder.ReleaseId;
+import org.kie.api.runtime.process.ProcessInstance;
+import org.kie.internal.runtime.conf.AuditMode;
+import org.kie.internal.runtime.conf.DeploymentDescriptor;
+import org.kie.internal.runtime.conf.ObjectModel;
+import org.kie.scanner.KieMavenRepository;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import static org.junit.Assert.*;
+import static org.kie.scanner.KieMavenRepository.getKieMavenRepository;
+
+public class ProcessServiceWithEntitiesTest extends AbstractKieServicesBaseTest {
+
+    private static final Logger logger = LoggerFactory.getLogger(ProcessServiceWithEntitiesTest.class);
+    private static final String CASEDETAIL_JAVA = "src/main/java/example/CaseDetail.java";
+    private List<DeploymentUnit> units = new ArrayList<DeploymentUnit>();
+
+    @Before
+    public void prepare() {
+        configureServices();
+        logger.debug("Preparing kjar");
+        KieServices ks = KieServices.Factory.get();
+        ReleaseId releaseId = ks.newReleaseId(GROUP_ID, ARTIFACT_ID, VERSION);
+        List<String> processes = new ArrayList<String>();
+        processes.add("repo/processes/general/entityprocessvar-process.bpmn2");
+
+
+        DeploymentDescriptorImpl customDescriptor = new DeploymentDescriptorImpl("org.jbpm.domain");
+        customDescriptor.setAuditMode(AuditMode.JPA);
+        customDescriptor.setEnvironmentEntries(null);
+
+        List<String> remotableClasses = java.util.Arrays.asList("example.CaseDetail", "org.drools.persistence.jpa.marshaller.MappedVariable");
+        customDescriptor.setClasses(remotableClasses);
+
+        List<ObjectModel> marshallingStrategies = new ArrayList<ObjectModel>();
+        marshallingStrategies.add(new ObjectModel("mvel", "new org.drools.persistence.jpa.marshaller.JPAPlaceholderResolverStrategy(\"org.jbpm.domain\", classLoader)"));
```

Comment:
shouldn't this use persistence unit named org.jbpm.test:test-module:1.0.0-SNAPSHOT instead of org.jbpm.domain?